test/Api.IntegrationTest/AdminConsole/Controllers/OrganizationUsersControllerPerformanceTests.cs

```diff
@@ -7,74 +7,33 @@
 
 namespace Bit.Api.IntegrationTest.AdminConsole.Controllers;
 
-public class OrganizationUsersControllerPerformanceTest : IClassFixture<ApiApplicationFactory>, IAsyncLifetime
+public class OrganizationUsersControllerPerformanceTest(ITestOutputHelper testOutputHelper)
 {
-    private readonly HttpClient _client;
-    private readonly ApiApplicationFactory _factory;
-    private readonly ITestOutputHelper _testOutputHelper;
-
-    public OrganizationUsersControllerPerformanceTest(ApiApplicationFactory factory, ITestOutputHelper testOutputHelper)
+    [Theory(Skip = "Performance test")]
+    [InlineData(100)]
+    [InlineData(60000)]
+    public async Task GetAsync(int seats)
     {
-        _factory = factory;
-        _testOutputHelper = testOutputHelper;
-        _client = _factory.CreateClient();
-    }
+        await using var factory = new ApiApplicationFactory();
```

Comment:
This is a much simpler way of handling this 👏

Reply:
Yea, I had to create a new factory per test instance since the data collides.